modules/update-process-workflow.adoc

```diff
@@ -27,6 +27,8 @@ The job then extracts the manifests and metadata from the release image to a sha
 Certain conditions can prevent updates from proceeding.
 These conditions are either determined by the CVO itself, or reported by individual cluster Operators that detect some details about the cluster that the Operator considers problematic for the update.
 
+// to do: potentially add an example of a precondition to the bullet above.
```

Comment:
This is in reference to [this discussion thread](https://github.com/openshift/openshift-docs/pull/60091#discussion_r1212416618).

Reply:
I think coming up with a specific example would be helpful, I'll try to provide one

Reply:
one possible option would be [`LowDesiredVersion`][1].

[1]: https://github.com/openshift/cluster-version-operator/pull/918/files#diff-1008d196a61fa8941a8df069ea3bd53275cc2f4e0b7d588529485db5598ed02fR58